quirks/password-rules.json

```diff
@@ -500,6 +500,9 @@
     "rejsekort.dk": {
         "password-rules": "minlength: 7; maxlength: 15; required: lower; required: upper; required: digit;"
     },
+    "renaud-bray.com": {
+        "password-rules": "minlength: 8; maxlength: 38; allowed: upper, lower, digits;"
```

Comment:
This isn’t a valid rule. Please use https://developer.apple.com/password-rules/ to craft a rule.